server/routes/bookedtoday/arrivals/searchforexisting/singleExistingRecordFoundController.ts

```diff
@@ -20,4 +33,25 @@ export default class SingleExistingRecordFoundController {
       })
     }
   }
+
+  public submit(): RequestHandler {
+    return async (req, res) => {
+      const { id } = req.params
+      const searchData = State.searchDetails.get(req)
+      const potentialMatches = await this.expectedArrivalsService.getMatchingRecords(searchData)
```

Comment:
it shouldn't be optional, by "passing the prison number"  did you mean rendering a hidden param and submitting as part of the form or including it as part of the url? 

Either way I think this is fine

Reply:
Yeah that was the plan 😁 Thought using the prisonNumber would be more accurate rather than researching on pnc, dob and names since potentially they could bring back different results if they've been modified in the mean time. But I suppose we're talking about edge edge cases here 😓 

Looks like it's optional on the [API side](https://github.com/ministryofjustice/hmpps-welcome-people-into-prison-api/blob/e8d8ea9f2769ff3a150759e366837d0f3db0d713/src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/arrivals/Arrival.kt#L66), so maybe that needs updating? 🤔